### PR TITLE
COS-268: Github actions

### PR DESCRIPTION
**Pull request by dmitry-shanko**

### CODE REVIEW

File: .github/workflows/attach-artifact-release.yml [hunk 1]
```diff
@@ -14,85 +14,99 @@ on:
...
-                  ASSET_DIR: ./assets
\ No newline at end of file
+            -   name: Import GPG key
+                id: import_gpg
+                uses: crazy-max/ghaction-import-gpg@v5
+                with:
+                    gpg_private_key: ${{ secrets.GPG_SECRET }}
+                    passphrase: ${{ secrets.GPG_PASSPHRASE }}
+
+            -   name: Sign Files for Draft Release
```
Comment (konstantin-umanets) [inline .github/workflows/attach-artifact-release.yml:97]: Do we need `Draft` here?

Comment (dmitry-shanko) [reply .github/workflows/attach-artifact-release.yml:97]: yes, release will be draft

File: .github/workflows/attach-artifact-release.yml [hunk 2]
```diff
@@ -14,85 +14,99 @@ on:
...
 
-            - name: Sign Files for Draft Release
-              run: |
-                  gpg -K
-                  version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
-                  ./.github/sign_artifact.sh ./assets/${{ env.artifact_id }}-${version}.jar
-                  ./.github/sign_artifact.sh ./assets/${{ env.artifact_id }}-${version}.pom
-                  ./.github/sign_artifact.sh ./assets/${{ env.artifact_id }}-${version}-javadoc.jar
-                  ./.github/sign_artifact.sh ./assets/${{ env.artifact_id }}-${version}-sources.jar
+            -   name: Delete Outdated Files from Draft Release
```
Comment (konstantin-umanets) [inline .github/workflows/attach-artifact-release.yml:80]: What's the source of extra spaces here between `-` and text? Is it a non-formatted old code or a different codestyle for yml files (or lack of formatting configuration for yml files)?

Comment (dmitry-shanko) [reply .github/workflows/attach-artifact-release.yml:80]: A very good question. I use default with .editorconfig and it somehow overrides default indent of 2 symbols.

**Review by konstantin-umanets — APPROVED** [review]

**Review by CountRick — APPROVED** [review]

**Pull request merged**